File: clients/javascript/src/models/cacheGetGroupIn.ts
// this file is @generated





export interface CacheGetGroupIn {
    name: string;
}

export const CacheGetGroupInSerializer = {
    _fromJsonObject(object: any): CacheGetGroupIn {
        return {
            name: object['name'],
            };
    },

    _toJsonObject(self: CacheGetGroupIn): any {
        return {
            'name': self.name,
            };
    }
}

File: clients/rust/src/models/cache_get_group_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[non_exhaustive]
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct CacheGetGroupIn {
    pub name: String,
}

impl CacheGetGroupIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
